twitter_ads/creative.py

```diff
@@ -456,27 +457,31 @@ class CardsFetch(Resource):
 
     PROPERTIES = {}
 
-    FETCH_URI = '/' + API_VERSION + '/accounts/{account_id}/cards'
+    FETCH_URI = '/' + API_VERSION + '/accounts/{account_id}/cards/all'
     FETCH_ID = '/' + API_VERSION + '/accounts/{account_id}/cards/all/{id}'
 
     def all(klass):
         raise AttributeError("'CardsFetch' object has no attribute 'all'")
 
-    def load(klass, account, card_uri=None, card_id=None, with_deleted=None):
+    def load(klass, account, card_uris=None, card_id=None, with_deleted=None):
         # check whether both are specified or neither are specified
-        if all([card_uri, card_id]) or not any([card_uri, card_id]):
-            raise ValueError('card_uri and card_id are exclusive parameters. ' +
+        if all([card_uris, card_id]) or not any([card_uris, card_id]):
+            raise ValueError('card_uris and card_id are exclusive parameters. ' +
                              'Please supply one or the other, but not both.')
         params = {}
-        if card_uri:
-            params['card_uri'] = card_uri
+        if with_deleted:
+            params['with_deleted'] = 'true'
+
+        if card_uris:
+            params['card_uris'] = ','.join(card_uris)
             resource = klass.FETCH_URI.format(account_id=account.id)
+            request = Request(account.client, 'get', resource, params=params)
+            return Cursor(klass.load, request, init_with=[account])
```

Comment:
@tushdante @juanshishido 
I slightly changed this line to pass `klass.load` as a first arg rather than just `klass`. Otherwise, I was getting:
> TypeError: 'CardsFetch' object is not callable
error with the updated example script I included this PR.

Let me know if this is wrong.

Reply:
hey @smaeda-ks  looks like the issue here is that you're missing the `@classmethod` decorator for the `load()` function. Could you try adding that and reverting `klass.load` to `klass`. That should fix the issue. Let me know if there's any other questions!

Reply:
Another change needed here:
* Change line 484 to `klass(account).from_response(response.body['data'])` since you need to instantiate the class to call the method (as opposed to passing in a reference to the class). This happens since we're setting the `load()` function for the `CardsFetch` class as a classmethod